addon/components/flash-message.js

```diff
@@ -73,6 +73,7 @@ export default Component.extend({
   willDestroy() {
     this._super();
     this._destroyFlashMessage();
+    run.cancel(this.pendingSet);
```

Comment:
Can we destructure `cancel`, then use `cancel(get(this, 'pendingSet'))` instead?